adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/impl/HttpBasedMessageMapping.java

```diff
@@ -121,6 +121,8 @@ private void mapMessageRequest(
                 headers.add(property.getKey(), Json.encode(value));
             }
         });
+        headers.add(MessageHelper.APP_PROPERTY_ORIG_ADDRESS, ctx.message().topicName());
+        headers.add(MessageHelper.SYS_PROPERTY_CONTENT_TYPE, ctx.contentType());
```

Comment:
Why don't we use the standard HTTP *Content-Type* header?

Reply:
you're right, the standard HTTP header is better, updated.